### DISCUSSION TITLE
I want to downgrade python in the base environment.

### DISCUSSION THREAD

**Discussion by okikosuke** [Homebrew/discussions#3443] [Everyday usage]
ENVIRONMENT
- anaconda
- windows 10

Currently the version of python in the base environment of anaconda is 3.9. I want to downgrade my python version to 3.8. How can I do this? I want to change the version of python in the base environment instead of creating a virtual environment.
I tried to change the version from anaconda navigation but it did not work. Also, in anaconda prompt

```python
conda install python=3.8

```
in anaconda prompt, but it did not work.

Translated with www.DeepL.com/Translator (free version)

**Comment by SMillerDev** [comment]
I'm not sure how to help you. Homebrew doesn't work on windows. Nor do I know anything about conda.
